operator/operator.make

```diff
@@ -12,3 +12,19 @@ _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION):
 # GOROOT is needed because of https://github.com/operator-framework/operator-sdk/issues/1854#issuecomment-525132306
 operator-generate-k8s: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION)
 	GOROOT=$(shell $(GO) env GOROOT) _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION) generate k8s
+
+operator-generate-crds: _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION)
```

Comment:
Should `deploy/crds/pmem-csi.intel.com_deployments_crd.yaml` be in the repo? If yes, commit it. If not, add it to .gitignore.

Reply:
Added to `deploy/crds` to .gitignore.